libs/dh/grid-areas/src/components/grid-areas.component.html

```diff
@@ -47,63 +43,54 @@ <h3>{{ t("topBarTitle") }}</h3>
         [options]="gridAreaStatusOptions"
         [multiple]="true"
         [chipMode]="true"
-        [(ngModel)]="selectedGridAreaStatus"
+        [(ngModel)]="selectedGridAreaStatuses"
         [placeholder]="t('statusTypePlaceholder')"
       />
     </vater-stack>
-
-    <vater-flex fill="vertical" scrollable>
-      <watt-table
-        [columns]="columns"
-        [dataSource]="dataSource"
-        aria-label="gridAreas"
-        sortBy="code"
-        sortDirection="asc"
-        [loading]="isLoading()"
-        [activeRow]="activeRow()"
-        (rowClick)="activeRow.set($event)"
-      >
-        <ng-container *wattTableCell="columns['code']; header: t('columns.code'); let gridArea">
-          {{ gridArea.code }}
-        </ng-container>
-
-        <ng-container *wattTableCell="columns['actor']; header: t('columns.actor'); let gridArea">
-          {{ gridArea.actor }}
-        </ng-container>
-
-        <ng-container
-          *wattTableCell="columns['organization']; header: t('columns.organization'); let gridArea"
-        >
-          {{ gridArea.organization }}
-        </ng-container>
-
-        <ng-container
-          *wattTableCell="columns['priceArea']; header: t('columns.priceArea'); let gridArea"
-        >
-          {{ gridArea.priceArea }}
-        </ng-container>
-
-        <ng-container *wattTableCell="columns['type']; header: t('columns.type'); let gridArea">
-          {{ t("types." + gridArea.type) }}
-        </ng-container>
-
-        <ng-container *wattTableCell="columns['status']; header: t('columns.status'); let gridArea">
-          <dh-gridarea-status-badge [status]="gridArea.status" />
-        </ng-container>
-      </watt-table>
-
-      @if (!isLoading() && (hasError() || dataSource.data.length === 0)) {
-        <vater-stack fill="vertical" justify="center">
-          <watt-empty-state
-            [icon]="hasError() ? 'custom-power' : 'cancel'"
-            [title]="hasError() ? t('error') : t('noGridAreas')"
-          />
-        </vater-stack>
-      }
-    </vater-flex>
-
-    <watt-paginator [for]="dataSource" [pageSize]="100" />
-  </vater-flex>
-</watt-card>
-
-<dh-grid-area-details [gridAreaId]="activeRow()?.id" (closed)="activeRow.set(undefined)" />
+  </watt-data-filters>
+
+  <watt-data-actions>
+    <watt-button icon="download" variant="text" (click)="download()">{{
+      "shared.download" | transloco
+    }}</watt-button>
+  </watt-data-actions>
+
+  <watt-table
+    *transloco="let resolveHeader; read: 'devExamples.processes.table.columns'"
```

Comment:
question: should the scope be `devExamples`?

suggestion:


```suggestion
    *transloco="let resolveHeader; prefix: 'devExamples.processes.table.columns'"
```

Reply:
Nope, to much copy paste